lib/mux/webhooks/crypto.ex

```diff
@@ -0,0 +1,14 @@
+defmodule Mux.Webhooks.Crypto do
+  @moduledoc false
+
+  def compute_signature(payload, secret) do
+    hmac(:sha256, secret, payload)
+    |> Base.encode16(case: :lower)
+  end
+
+  if Code.ensure_loaded?(:crypto) and function_exported?(:crypto, :mac, 4) do
+    defp hmac(digest, key, payload), do: :crypto.mac(:hmac, digest, key, payload)
+  else
+    defp hmac(digest, key, payload), do: :crypto.hmac(digest, key, payload)
+  end
+end
```

Comment:
I'm not sure we would want to expose a new module here `Mux.Webhooks.Crypto`.

Unless I'm missing something,  it feels like it would be better to keep functionality in TestUtils since it is intended for that purpose and outside the scope of using this library in application code.

Reply:
I removed the `Mux.Webhooks.Crypto` module in favor of moving the calls to `hmac/3` local to the modules.